docs/how-to-guides/integrating-autoware-with-your-vehicle.md

```diff
@@ -170,4 +170,76 @@ Now the vehicle should drive along the calculated path!
 
 You may need to tune your parameters depending on the domain in which you will operate your vehicle.
 
+## 8. Customize your own package based on Autoware-msgs / Autoware-package
+
+In many practical applications, apart from the available nodes / modules of Autoware, you may have the need to create your own packages which communicate with Autoware nodes or utilize some Autoware implementations (such like math methods including A-star, interpolation, mpc algorithm and so on). In this case, you can follow the instructions below to customize your specific package.
+
+### Package using Autoware-msgs
+
+Since Autoware is built on ROS (Autoware.Universe / Autoware.Core on ROS2), if you have the urge to communicate with other Autoware nodes, then you are supposed to obey the rule of node subscribing / publishing messages via topic in specified message type. Please refer to the [ROS Tutorial](https://docs.ros.org/en/humble/Tutorials.html) for details.
+
+If you are already experienced at ROS, then it's simple to do such an extension just like the following example. Here, as mentioned in section 5.1 above, how could vehicle interface package (such as driving-by-wire module) receives the control command? You can do:
+
+- Put the "autoware_auto_control_msgs" in your project with your own packages together
+- Add the "depend" tag in "package.xml" of your package which receives the control command
+
+```shell
+<depend>autoware_auto_control_msgs</depend>
+```
+
+- Add message path in "CMakeLists.txt"
+
+```shell
+find_package(autoware_auto_control_msgs)
+```
+
+- Include the header file of the message type and start coding
+
+```shell
+#include <autoware_auto_control_msgs/msg/ackermann_control_command.hpp>
+```
```

Comment:
Please use the correct types.

``````suggestion
```xml
<depend>autoware_auto_control_msgs</depend>
```

- Add message path in "CMakeLists.txt"

```cmake
find_package(autoware_auto_control_msgs)
```

- Include the header file of the message type and start coding

```cpp
#include <autoware_auto_control_msgs/msg/ackermann_control_command.hpp>
```
``````